package.json

```diff
@@ -132,6 +132,7 @@
     "@types/tinycolor2": "1.4.2",
     "@typescript-eslint/eslint-plugin": "4.22.0",
     "@typescript-eslint/parser": "4.22.0",
+    "@welldone-software/why-did-you-render": "6.1.1",
```

Comment:
I'm ok with keeping it as a dev dependency, but we should probably document this somewhere so people can use it frecuently and we don't end up having multiple measurements around.

Reply:
Actually let me remove it for now. It was helpful but a little heavyweight. I'd prefer to introduce something like that in a separate PR.